otlp/trace.go

```diff
@@ -20,17 +19,14 @@ const (
 	traceIDLongLength  = 16
 )
 
-func TranslateHttpTraceRequest(req *http.Request) ([]map[string]interface{}, error) {
-	reqInfo := GetRequestInfoFromHttpHeaders(req)
-	if !reqInfo.HasValidContentType() {
-		return nil, ErrInvalidContentType
+func TranslateHttpTraceRequest(body io.ReadCloser, ri RequestInfo) ([]map[string]interface{}, error) {
```

Comment:
With the interface change here, the README ought to be updated.

Also, making a note at least for myself, that the downstream projects will need to update their uses of this function and `parseOTLPBody()` when they come to use this version.

Reply:
Yeah, SecureTenancy will need to update to take the next version of Husky.